charts/datadog/values.yaml

```diff
@@ -53,6 +53,16 @@ datadog:
     # datadog.secretBackend.timeout -- Configure the secret backend command timeout in seconds.
     timeout:  # 30
 
+    # datadog.secretBackend.enableGlobalPermissions -- Whether to create a global permission allowing Datadog agents to read all secrets
+    enableGlobalPermissions: true
```

Comment:
To avoid confusion, let's mention that this only works when `datadog.secretBackend.command` is set to `/readsecret_multiple_providers.sh`.

Reply:
Done